stl/inc/iosfwd

```diff
@@ -136,6 +136,11 @@ private:
 
 using streampos  = fpos<_Mbstatet>;
 using wstreampos = streampos;

```

Comment:
Nitpick: the "conventional" ordering is 
* char
* char8_t
* char16_t
* char32_t
* wchar_t

Reply:
I've approved, because I mildly prefer this ordering with "characters with native/locale-dependent encoding" before "characters with fixed encodings" - Yes, despite that `wchar_t` is fixed in practice - but I certainly wouldn't object to reordering.

Reply:
Same here, I just cannot resist to ring the "inconsistency" bell